setup.cfg

```diff
@@ -94,7 +94,7 @@ install_requires =
     environ-config >= 19.1.0
     envparse
     gidgethub >= 4.2.0
-    pyjwt[crypto]
+    pyjwt[crypto] >=2.0.0
```

Comment:
I'll probably do a conditional instead to support the old version too. I don't see a compelling reason to drop support for it since the compatibility needs just a one-line fix.

Reply:
ack, i have changed the method to support both with the conditional check.

Reply:
Oh, I actually have a local change for this. Also, yours fails the linting.

Reply:
oh ok, i see you added your changes
would you like me to remove my changes or how would you like to proceed with this.

Reply:
I'll just finish it myself. I've got a test in mind that I'm working on. I'll merge this once I'm happy with that.